client/controllers/bookHandler/bookHandler.go

```diff
@@ -0,0 +1,25 @@
+package controllers
+
+import (
+	"context"
+
+	pb "github.com/PavelDonchenko/bookstorejRPC/client/gen/proto"
+)
+
+type BaseBookHistoryHandler struct {
```

Comment:
How about `BookHistory`?

Reply:
If you mean to rename the struct, as for me BookHistory very generic word